common/src/main/java/org/duracloud/mill/config/ConfigurationManager.java

```diff
@@ -31,9 +31,88 @@ public String getWorkDirectoryPath() {
         return getCommaSeparatedListToArray(ConfigConstants.NOTIFICATION_RECIPIENTS);
     }
 
+    /**
+     * @return
+     */
+    public String[] getSwiftConfig() {
+        String[] config = new String[] {
+            System.getProperty(ConfigConstants.SWIFT_ACCESS_KEY),
+            System.getProperty(ConfigConstants.SWIFT_SECRET_KEY),
+            System.getProperty(ConfigConstants.SWIFT_ENDPOINT),
+            System.getProperty(ConfigConstants.SWIFT_SIGNER_TYPE)
+        };
+        return config;
+    }
+
+    /**
+     * @return
+     */
+    public String getQueueType() {
+        String queueType = System.getProperty(ConfigConstants.QUEUE_TYPE).trim();
+        if ( queueType.equalsIgnoreCase("aws") ) {
+            return "AWS";
+        } else if ( queueType.equalsIgnoreCase("rabbitmq") ) {
+            return "RABBITMQ";
+        } else {
+            return "Unknown";
+        }
+    }
+
+    /**
+     * @return
+     */
+    public String[] getRabbitMQConfig() {
+
+        if ( getQueueType() == "RABBITMQ" ) {
```

Comment:
Fixed in !674fd7b (although using `.equals`, because this value is hard coded (and formatted nicely because it's used for logging). Will make all these constants in an upcoming commit.